package.json

```diff
@@ -255,6 +256,7 @@
 		"lint:css": "stylelint \"**/*.scss\" --syntax scss",
 		"lint:js": "npm run -s install-if-deps-outdated && eslint --ext .js --ext .jsx --ext .ts --ext .tsx --cache .",
 		"lint:mixedindent": "mixedindentlint --ignore-comments \"client/**/*.scss\" \"assets/**/*.scss\" \"**/*.js\" \"**/*.jsx\" \"**/*.tsx\" \"**/*.tsx\" \"!build/**\" \"!node_modules/**\" \"!public/**\" \"!client/config/index.js\"",
+		"lint:package-json": "npmPkgJsonLint ./package.json ./packages/**/package.json ./apps/**/package.json",
```

Comment:
You probably should wrap patterns with quotes to prevent the default behaviors of CLIs. When using ZSH, it will convert those patterns into a list of matched files. I guess it's still correct but each env can behave differently.

Reply:
yup! https://github.com/Automattic/wp-calypso/pull/36534/commits/3b409a5de1eb97b64dc07279b4caed04dd6f17ce